### DISCUSSION TITLE
Git blame

### DISCUSSION THREAD

**Discussion by cajone** [NeogitOrg/neogit#1588] [Q&A] [answered]
I cannot find any reference in the manual on how I would be able to do a 'git blame' panel against the current buffer. I''m sure this must be available somewhere I'm just not finding it. Actually I cannot find any reference to the word 'blame' in the manual.

On further digging I did find some references to Git blame functionality by using a separate plugin

**Comment by CKolkey** [comment, marked as answer]
Yeah, haven't implemented blame yet, so for now you'll need a separate plugin.
